Libraries/Components/ViewPager/ViewPagerAndroid.android.js

```diff
@@ -97,17 +97,17 @@ var ViewPagerAndroid = React.createClass({
     ]),
   },
 
-  getInitialState: function() {
-    return {
-      selectedPage: this.props.initialPage,
-    };
+  componentDidMount: function() {
+    if (this.props.initialPage) {
+      this.setPageWithoutAnimation(this.props.initialPage);
+    }
   },
 
-  getInnerViewNode: function() {
+  getInnerViewNode: function(): ReactComponent {
     return this.refs[VIEWPAGER_REF].getInnerViewNode();
   },
 
-  _childrenWithOverridenStyle: function() {
+  _childrenWithOverridenStyle: function(): Array {
```

Comment:
Good question. Try `Array<ReactElement>` and run `flow` from the root of the repo and see if that works.

Doesn't look like the React codebase uses Flow so not sure how the Flow handles the boundaries between code that uses Flow and code that doesn't use it.

If `Array<ReactElement>` doesn't work I'd say `Array` or `Array<Object>` are perfectly fine - it's a private function you're using just in this file.